week2/materials/homework/assignments.md

```diff
@@ -4,51 +4,86 @@
 
 Sync your changes to s3 and write down your s3 bucket url.
 
-bucket url: `?`
+bucket url: `http://jean-claude-week2.s3-website-us-east-1.amazonaws.com`
 
 **Assignment 5:**
 
 Right now, the website does not support https. Explain which other AWS service needs to be integrated in order to achieve this.
 
-service name: `?`
```

Comment:
AWS Certificate Manager is indeed a service for creating certificates to be used securing web protocols like https. 

The service is wanted you to write though is called cloudfront, which can also help you achieve this (and a bunch of other things). 

See more here: https://aws.amazon.com/cloudfront/